org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/dashboard/DPRecentItems.java

```diff
@@ -314,6 +314,7 @@ private void removeLink(A btn) {
 			riDBremove(AD_RecentItem_ID);
 			bxRecentItems.removeChild(btn);
 			bxRecentItems.invalidate();
+			refresh();

```

Comment:
For better UI responsiveness, should call refresh(new ServerPushTemplate(bxRecentItems.getDesktop()) instead.

Reply:
> For better UI responsiveness, should call refresh(new ServerPushTemplate(bxRecentItems.getDesktop()) instead.

Thanks @hengsin - changed there and also in line 248